code/modules/mob/living/simple_animal/familiars/familiars.dm

```diff
@@ -45,6 +45,8 @@
 	icon = 'icons/mob/simple_animal/spaceshark.dmi'
 	pixel_x = -16
 
+	offset_overhead_text_x = 16
```

Comment:
I assume you didn't mean to include the pike commit?

Reply:
I think when I originally wrote this comment you hadn't mentioned it and then when I posted it in the morning you had. Sorry! Yeah, it's fine.